src/popup/components/InputField.vue

```diff
@@ -159,10 +163,16 @@ export default {
   },
   methods: {
     checkIfNumber(event) {
-      if (this.type === 'number' && !['Enter', 'Backspace'].includes(event.key) && !/^[0-9/-]$/.test(event.key)) {
+      if (!(this.type !== 'number' || event.key.length > 1 || ['ctrlKey', 'metaKey'].some((key) => event[key]))
```

Comment:
With the multiline if statements I suggest to do the same approach as with the object properties. So we are able to do this:
```
{ a: 1, b: 2, c: 3 }
```
and this:
```
{
  a: 1,
  b: 2,
  c: 3
}
```
Nothing in between, so this is wrong:
```
{ a: 1, b: 2,
c: 3 }
```
I think the same rule should be applied to if statements. So I suggest to do it as follows:
```
if (
  (test1 || test2 || test3)
  && (test4 || test5)
  && test6
) {
  ...
}
```